### PR TITLE
additional footsteps for certain kinds of entities

### PR DESCRIPTION
**Pull request by TiberianErin**
something from your nightmares is rapidly approaching

#### Summary
Content "Additional footstep types for certain types of entities"

#### Purpose of change

I noticed some variety that was never needed in the first place lacking. So I added more footsteps to more entity types.

#### Describe the solution

It adds additional footsteps to the game.

#### Describe alternatives you've considered

There arent any that I could think of, other than the player just pretending the footstep sounds are not what the game says they sound like.

#### Testing

Starting game after adding updated file. Custom Character -> Freeform
Adding good hearing and starting game.
Used debug menu to add Fey Hearing (best hearing i can get)
Standing behind wall of evac shelter
Spawned jabberwock on the other side.
success
horror and abhorrent footsteps work.
complete - it works

Knowing I used the same technique for all footsteps, i am assuming that this PR is bugless.

#### Additional context

Changed footsteps for:
- Horror
- Aberration
- Unknown (footsteps to differentiate between ~~lazy~~ forgetful / rushed contributors who forgot to put a category on their mobs and those who remembered to do so)

https://youtu.be/gHEYe-WwlSo

### CODE REVIEW

File: data/json/species.json [hunk 1]
```diff
@@ -182,6 +184,7 @@
   {
     "type": "SPECIES",
     "id": "UNKNOWN",
-    "description": "a bug"
+    "description": "a bug",
+    "footsteps": "bugs"
```
Comment (Maleclypse) [inline data/json/species.json:188]:
```suggestion
    "footsteps": "clisp, clisp, clisp, 'faster then slower'"
```

**Review by Maleclypse — COMMENTED** [review]
What do you think?

File: data/json/species.json (hunk 1 above)
Comment (TiberianErin) [reply data/json/species.json]: "clisp"?
is that what errors make?

**Comment by TiberianErin** [comment]
> What do you think?

a lot of these are good, i'm just confused about the "clisp" stuff

**Comment by Maleclypse** [comment]
> > What do you think?
> 
> a lot of these are good, i'm just confused about the "clisp" stuff

I listed to a youtube video of super powerful microphones listening to various bugs walking and that was the closest I could come to recreating the sound centipedes made.

**Comment by TiberianErin** [comment]
> > > What do you think?
> > 
> > 
> > a lot of these are good, i'm just confused about the "clisp" stuff
> 
> I listed to a youtube video of super powerful microphones listening to various bugs walking and that was the closest I could come to recreating the sound centipedes made.

my main two problems:
1 - it needs to be obviously a bug
2 - the checks need to be not failed (one of them failed)

**Comment by TiberianErin** [comment]
what is causing the failing check?

**Comment by NetSysFire** [comment]
A bug that was already fixed in #62405. Rebase if it annoys you, or just ignore it since it is not a required check.

**Comment by TiberianErin** [comment]
> A bug that was already fixed in #62405. Rebase if it annoys you, or just ignore it since it is not a required check.

guess i'll ignore it?

**Comment by TiberianErin** [comment]
bump because it's been forever

**Comment by NetSysFire** [comment]
DAMN YOU STALEBOT. Also I think merging was toned down until 0.G I believe.

**Comment by Maleclypse** [comment]
> DAMN YOU STALEBOT. Also I think merging was toned down until 0.G I believe.

If you see something that’s frozen and shouldn’t be hit with stalebot please add 0.G String Freeze label on it.

**Comment by Fris0uman** [comment]
Those deviate pretty significantly from other footstep. All footstep are one word long description of the sound, those are much longer and I think it breaks the pattern and looks a bit weird

**Comment by kevingranade** [comment]
Fris' comments are pretty critical,  the tone of these messages need to match.

**Pull request closed without merging**